sdk/python/pulumi_kubernetes/provider.py

```diff
@@ -0,0 +1,38 @@
+import pulumi
+from . import tables
+
+class Provider(pulumi.ProviderResource):
+    """
+    The provider type for the kubernetes package.
+    """
+    def __init__(self,
+                 __name__,
+                 __opts__=None,
+                 cluster=None,
+                 context=None,
+                 kubeconfig=None,
+                 namespace=None):
+        """
+        Create a Provider resource with the given unique name, arguments, and options.
+
+        :param str __name__: The unique name of the resource.
+        :param pulumi.ResourceOptions __opts__: An optional bag of options that controls this resource's behavior.
+        :param str cluster: If present, the name of the kubeconfig cluster to use.
+        :param str context: If present, the name of the kubeconfig context to use.
+        :param str kubeconfig: The contents of a kubeconfig file. If this is set, this config will be used instead
+                               of $KUBECONFIG.
+        :param str namespace: If present, the namespace scope to use.
+        """
+        __props__ = {
+            "cluster": cluster,
+            "context": context,
+            "kubeconfig": kubeconfig,
+            "namespace": namespace,
+        }
+        super(Provider, self).__init__("kubernetes", __name__, __props__, __opts__)
+
+    def translate_output_property(self, prop: str) -> str:
+        return tables._CASING_FORWARD_TABLE.get(prop) or prop
```

Comment:
What is the effect of this on this resource? It _should_ be nothing, right? If so, should it be omitted?

Reply:
You're right, it "should" be nothing, and we don't really need it. I'll kill it.